Build error on host without tcpdump

Hi,

I updated the tcpreplay version in buildroot[1] to 4.0.0 (will update to 4.0.2 shortly). While it built fine for me and others the automated build system started showing failures[2]. Eventually another developer figured out that if the host system didn't have tcpdump installed then the build would fail. Reproduction below
## 

[1] - http://buildroot.org
[2] - http://autobuild.buildroot.net/results/69e/69e5adcb5c92259566556aea2b0ac4290f6d0cd2/build-end.log

```
sudo mv /usr/sbin/tcpdump /usr/sbin/tcpdump.copy
./configure
make
...
gcc -DHAVE_CONFIG_H -I.    -I../libopts -I..   -DTCPREPLAY -D_U_="__attribute__((unused))"  -Wall -O3 -std=gnu99  -Wno-variadic-macros -Wfatal-errors -Wno-format-contains-nul -I/usr/include -MT tcpreplay-tcpreplay_api.o -MD -MP -MF .deps/tcpreplay-tcpreplay_api.Tpo -c -o tcpreplay-tcpreplay_api.o `test -f 'tcpreplay_api.c' || echo './'`tcpreplay_api.c
tcpreplay_api.c: In function ‘tcpreplay_set_verbose’:
tcpreplay_api.c:693:5: error: expected expression before ‘)’ token
```

--- a/src/tcpreplay_api.c
+++ b/src/tcpreplay_api.c
@@ -690,7 +690,7 @@ tcpreplay_set_verbose(tcpreplay_t *ctx, bool value)
     ctx->options->verbose = value;
     return 0;
 #else
-    tcpreplay_seterr(ctx, "verbose mode not supported");
+    tcpreplay_seterr(ctx, "%s", "verbose mode not supported");
     return -1;
 #endif
 }
@@ -710,7 +710,7 @@ tcpreplay_set_tcpdump_args(tcpreplay_t *ctx, char *value)
     ctx->options->tcpdump_args = safe_strdup(value);
     return 0;
 #else
-    tcpreplay_seterr(ctx, "verbose mode not supported");
+    tcpreplay_seterr(ctx, "%s", "verbose mode not supported");
     return -1;
 #endif
 }
@@ -731,7 +731,7 @@ tcpreplay_set_tcpdump(tcpreplay_t *ctx, tcpdump_t *value)
     ctx->options->tcpdump = value;
     return 0;
 #else
-    tcpreplay_seterr(ctx, "verbose mode not supported");
+    tcpreplay_seterr(ctx, "%s", "verbose mode not supported");
     return -1;
 #endif
 }

